gems/sorbet-runtime/test/types/edge_cases.rb

```diff
@@ -117,6 +122,11 @@ def foo(x=:foo)
         assert_raises(TypeError) do
           klass.new.bar(1)
         end
+
+        # Should use fast path
+        obj = klass.new
+        allocs = counting_allocations {obj.bar}
+        assert(allocs < 5)
```

Comment:
Do you have a bound for the number of allocations done on the slow path? Mostly just curious to know what the difference between the two is.

Reply:
It's 20-30 on the slow path (varies a bit by exact case), 3 on the fast path.